### ISSUE TITLE
Fallback KV doesn't actually do anything because HeaderlessKVBufferToKeyValues doesn't return null on failed parsing

### ISSUE THREAD

**Issue by khang06** [Joshua-Ashton/VPhysics-Jolt#43]
I can't find a single case where `KeyValues::LoadFromBuffer` returns false. To prove this further, I stuck this line in:
![image](https://user-images.githubusercontent.com/11239786/187560318-0480c002-9b15-43b9-91c5-8b1049907f3c.png)

Even if parsing fails, it just returns an object anyway. Not sure if its state is corrupt or not since I don't usually do this kind of thing.
![image](https://user-images.githubusercontent.com/11239786/187560368-4218349e-8e82-4fcc-9d63-e96bf2150056.png)

**Comment by Joshua-Ashton** [comment]
This might be something that differs per-Source branch.
Are compiling for SDK 2013? I was getting NULL back for GMod builds.

**Comment by khang06** [comment]
Yeah, this is SDK 2013 SP. I haven't checked on MP but I don't think there's any reason for it to be different on that.
